.github/workflows/doc-build.yml

```diff
@@ -89,7 +89,7 @@ jobs:
           DOC_TARGET="html"
         fi
 
-        DOC_TAG=${DOC_TAG} SPHINXOPTS="-q -W -t publish" make -C doc ${DOC_TARGET}
```

Comment:
We have no warnings upstream, so why is this required?

Reply:
you can see the discussion about it here: NCSDK-16980

Reply:
done